lib/backdrop.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+
+class BackDrop extends StatelessWidget {
+  BackDrop({super.key});
+
+  String backDropImagePath = 'assets/images/backdropImage.jpg';
+
+  @override
+  Widget build(BuildContext context) {
+    return Image.asset(backDropImagePath);
+  }
+}
```

Comment:
This abstraction is not necessary you can directly use Image.asset instead to create a widget to do that